proj/cmake/modules/cinderMakeApp.cmake

```diff
@@ -61,12 +61,42 @@ function( ci_make_app )
 		set_source_files_properties( ${ARG_RESOURCES} PROPERTIES HEADER_FILE_ONLY ON MACOSX_PACKAGE_LOCATION Resources )
 	elseif( CINDER_LINUX )
 		unset( ARG_RESOURCES ) # Don't allow resources to be added to the executable on linux
+	elseif( CINDER_MSW )		
+		if( MSVC )
+			# Override the default /MD with /MT
+			foreach( 
+				flag_var
+				CMAKE_C_FLAGS CMAKE_C_FLAGS_DEBUG CMAKE_C_FLAGS_RELEASE CMAKE_C_FLAGS_MINSIZEREL CMAKE_C_FLAGS_RELWITHDEBINFO 
+				CMAKE_CXX_FLAGS CMAKE_CXX_FLAGS_DEBUG CMAKE_CXX_FLAGS_RELEASE CMAKE_CXX_FLAGS_MINSIZEREL CMAKE_CXX_FLAGS_RELWITHDEBINFO 
+			)
+				if( ${flag_var} MATCHES "/MD" )
+					string( REGEX REPLACE "/MD" "/MT" ${flag_var} "${${flag_var}}" )
+					set( "${flag_var}" "${${flag_var}}" PARENT_SCOPE )
+				endif()
+			endforeach()
+			# Force synchronous PDB writes
+			add_compile_options( /FS ) 
+			# Force multiprocess compilation
+			add_compile_options( /MP )
+			# Add necessary platform libraries		
+			list( APPEND PLATFORM_LIBRARIES "zlib.lib;shlwapi.lib;OpenGL32.lib" )
```

Comment:
On OS X, we handled this as part of libcinder's `target_link_libraries()` ([here](https://github.com/cinder/Cinder/blob/524c705c36adbf9644209d2210dc7dec26685f98/proj/cmake/platform_macosx.cmake#L87-L97)) Its pretty nice this way because then anyone who does a `find_package( cinder )` will get linked to the required libraries, even if they aren't using `ci_make_app()` (like if they already have a different cmake setup).